OnDemandPassengerUITests/OnDemandPassengerUITests.swift

```diff
@@ -0,0 +1,50 @@
+
+
+//
+//  OnDemandPassengerUITests.swift
+//  OnDemandPassengerUITests
+//
+//  Created by Killion, Connor on 5/26/17.
+//  Copyright © 2017 HERE. All rights reserved.
+//
+
+import XCTest
+
+class OnDemandPassengerUITests: XCTestCase {
```

Comment:
We use Quick framework for our unit tests you can get ViewControllerTests as an example.